### PR TITLE
Add pinout and environments for ESP32S3 Xiao Sense boards

### PR DESCRIPTION
**Pull request by lorow**
Adds pinout and environment settings for the Seedstudio Xiao Sense ESP32S3 boards according to the pinout given on their wiki: https://wiki.seeedstudio.com/xiao_esp32s3_camera_usage/

### CODE REVIEW

**Comment by lorow** [comment]
https://github.com/limengdu/SeeedStudio-XIAO-ESP32S3-Sense-camera/blob/main/CameraWebServer/camera_pins.h

Thanks to lmore377#0001

**Comment by Blu3u** [comment]
I had the same issue before with ESPS3WROOM. It's too easy to miss that.

**Comment by ZanzyTHEbar** [comment]
I have ordered a XIAO board for testing. Will be here soon :)

File: ESP/lib/src/data/config/project_config.cpp [hunk 1]
```diff
@@ -307,7 +309,7 @@ void ProjectConfig::setWifiConfig(const std::string& networkName,
 
   if (shouldNotify) {
     wifiStateManager.setState(WiFiState_e::WiFiState_None);
-    WiFi.disconnect();
+    //WiFi.disconnect();
```
Comment (EtheriVR) [inline ESP/lib/src/data/config/project_config.cpp:312]: Leaving these commented out makes it potentially look like a mistaken commit?

File: ESP/lib/src/network/wifihandler/wifihandler.cpp [hunk 2]
```diff
@@ -8,23 +8,32 @@ WiFiHandler::WiFiHandler(ProjectConfig& configManager,
...
+
+  log_d(
+      "ADHOC is disabled, setting up STA network and checking transmission "
+      "power \n\r");
   auto txpower = configManager.getWiFiTxPowerConfig();
-  WiFi.mode(WIFI_STA);
-  WiFi.setSleep(WIFI_PS_NONE);
+  log_d("Setting Wifi Power to: %d", txpower.power);
+  // log_d("Enabling STA mode \n\r");
+  // WiFi.mode(WIFI_STA);
```
Comment (EtheriVR) [inline ESP/lib/src/network/wifihandler/wifihandler.cpp:34]: Leaving these commented out makes it potentially look like a mistaken commit?

File: ESP/src/main.cpp [hunk 3]
```diff
@@ -84,14 +78,12 @@ void setup() {
   setCpuFrequencyMhz(240);
   Serial.begin(115200);
   Logo::printASCII();
-  Serial.flush();
+  //  Serial.flush();
```
Comment (EtheriVR) [inline ESP/src/main.cpp:81]: Leaving these commented out makes it potentially look like a mistaken commit?

**Pull request merged**